app/models/piece.rb

```diff
@@ -3,6 +3,20 @@ class Piece < ActiveRecord::Base
 
   enum color: [:white, :black]
 
+  def move_to!(new_x, new_y)
+    piece = Game.find(game_id).pieces.find_by(x_coordinate: new_x, y_coordinate: new_y)
+    if !piece.nil?
+      if piece.color == color
+        'Error'
+      elsif piece.color != color
```

Comment:
On this line, you can use `else`. No need for `elsif...` since there are only 2 possibilities